apps/framework-docs/src/pages/quick-reference.mdx

```diff
@@ -0,0 +1,341 @@
+import { Tabs } from "nextra/components";
+
+# Quick Reference
+
+This is a quick guide on using MooseJS with all the basic code in one place. For more details you can visit individual pages on each section.
+
+---
+
+<Tabs items={["Data Models", "Flows", "Aggregations", "Consumption APIs"]}>
+
+    <Tabs.Tab>
+        ## Creating a Model
+
+        To create a data model you need to open the `models.ts` file in your project's `datamodels` directory
+        and add a new Data Model definition.
+
+        ```ts filename="datamodels/models.ts" copy
+        import { Key } from "@514labs/moose-lib";
+
+        export interface UserActivity {
+        eventId: Key<string>;
+        timestamp: string;
+        userId: string;
+        activity: string;
```

Comment:
Looks like you could add some indentation to those code blocks to make them render better